install/concurrency/docs/index.html

```diff
@@ -1,26 +1,14 @@
-<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
 <html>
-<head>
-<!--
-
-    Copyright (c) 2018, 2020 Oracle and/or its affiliates. All rights reserved.
-
-    This program and the accompanying materials are made available under the
-    terms of the Eclipse Public License v. 2.0, which is available at
-    http://www.eclipse.org/legal/epl-2.0.
-
-    This Source Code may also be made available under the following Secondary
-    Licenses when the conditions for such availability set forth in the
-    Eclipse Public License v. 2.0 are satisfied: GNU General Public License,
-    version 2 with the GNU Classpath Exception, which is available at
-    https://www.gnu.org/software/classpath/license.html.
-
-    SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
-
--->
-
-  <title>Welcome to the Concurrency TCK, Version 2.0</title>
-  <style type="text/css">
+  <head>
+    <meta http-equiv="content-type" content="text/html; charset=windows-1252">
+    <!--
+    Copyright (c) 2018, 2020 Oracle and/or its affiliates. All rights reserved.    This program and the accompanying materials are made available under the
+    terms of the Eclipse Public License v. 2.0, which is available at    http://www.eclipse.org/legal/epl-2.0.    This Source Code may also be made available under the following Secondary
+    Licenses when the conditions for such availability set forth in the    Eclipse Public License v. 2.0 are satisfied: GNU General Public License,    version 2 with the GNU Classpath Exception, which is available at    https://www.gnu.org/software/classpath/license.html.
+    SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0-->
+    <title>Welcome to the Concurrency TCK, Version 2.0</title>
```

Comment:
As we have updated the TCK version for Standalone TCK's, in that new Concurrency TCK version is updated as 2.0.1, I guess we might need to update this from Version 2.0 to Version 2.0.1

Reply:
While reviewing the rest of the user guide changes, I see the version is limited up to two digits, for example, 2.0 or 3.0, etc. So I believe this is what it supposed to be. Still, I would like to hear more from experts.

Reply:
The Spec version for Concurrency is 2.0 as is shown on https://jakarta.ee/specifications/concurrency.  Any two digit version number such as `2.0` is referring to the SPEC version, not the TCK version.  

In summary, the individual SPEC versions should not be changed for Jakarta EE 9.1, as the SPECs are not being released again.  So, please do update Concurrency TCK 2.0.0 references to the new TCK version, but do not change the Concurrency SPEC version which should stay 2.0.

Reply:
I removed the version number from the title element to match the others.